upi/metal/main.tf

```diff
@@ -8,8 +8,6 @@ locals {
 
     # "rd.break=initqueue"
     "coreos.inst=yes",
-
-    "coreos.inst.image_url=${var.pxe_os_image_url}",
     "coreos.inst.install_dev=sda",
     "coreos.inst.skip_media_check",
```

Comment:
Also a no-op.

Reply:
Right, this is only here for now until we land https://github.com/openshift/release/pull/10121